src/index.ts

```diff
@@ -7,6 +7,7 @@ export { addSecond } from "./addSecond"
 export { ap } from "./ap"
 export { applyOffset } from "./applyOffset"
 export { date } from "./date"
+export * from "./duration"
```

Comment:
We need to use explicit exports for the build process to properly optimize these:

```suggestion
export { formatOrParseDuration, parseDuration, formatDuration } from "./duration"
```

Reply:
Understood, Justin. I'll make the suggested changes to use explicit exports to ensure proper optimization during the build process. Thank you for pointing that out.